### DISCUSSION TITLE
Moving `probot/settings` to `repository-settings/app`

### DISCUSSION THREAD

**Discussion by gr2m** [repository-settings/app#519] [Announcements]
We are planning on moving the repository for the [settings app](https://github.com/apps/settings) to its own, dedicated organization: @repository-settings.

**There will be no change for existing users.**

Together with the the change of organization we will also change the deployment of the app. It currently is deployed to a legacy Heroku server owned by GitHub. GitHub will shut down this server as nobody within GitHub is maintaining the settings app.

We will instead deploy the app as a serverless app to Vercel. We will continue to send exceptions to [Sentry](https://sentry.io/welcome/) and will send logs to [Logflare](https://logflare.app/), which will enable us to build a public dashboard such as the one I built for [WIP](https://wip.vercel.app/stats) and [all-contributors](https://all-contributors.vercel.app/).

Nothing else will change. @travi is already the defacto maintainer of the settings app for a long time, this step will make it official. I will help out as a co-maintainer.

Please let us know if you have any questions or concerns.

**Comment by mattwynne** [comment]
Hi,

I just wondered how this is going, and if there was anything I could do to help. We're starting to adopt this app to manage settings for https://github.com/cucumber org, and I'm keen to understand whether we'll be able to contribute to the app and get those changes into production.

**Reply by mattwynne** [reply]
For example, I'm super interested in seeing https://github.com/probot/settings/issues/518 come to life.
